### PR TITLE
django 1.8 fix errors. disable status for staff users only

### PR DESCRIPTION
**Pull request by kalombos**
Fixed errors for django 1.8. The status is used for not-staff users now. Changed responses for form view.

### CODE REVIEW

File: flag/models.py [hunk 1]
```diff
@@ -121,7 +121,7 @@ class FlaggedContent(models.Model):
                                   null=True,
                                   related_name="moderated_content")
     count = models.PositiveIntegerField(default=0)
-    when_updated = models.DateTimeField(auto_now=True, auto_now_add=True)
```
Comment (gEndelf) [inline flag/models.py:124]: @kalombos why did you remove that

Comment (kalombos) [inline flag/models.py:124]: @gEndelf because i got the error 
flag.FlaggedContent.when_updated: (fields.E160) The options auto_now, auto_now_add, and default are mutually exclusive. Only one of these options may be present.
